app/workers/expired_rent_scheduler.rb

```diff
@@ -0,0 +1,13 @@
+require 'sidekiq-scheduler'
+require 'date'
+
+class ExpiredRentScheduler
+  include Sidekiq::Worker
+
+  def perform
+    rents = Rent.where('final_date < ?', Time.zone.today)
```

Comment:
Are you realy sure of this query? I am not sure you are retrieving the right rents with that.